### DISCUSSION TITLE
aprenda com o "website do cara***"

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#179] [Discussão]
vocês meus caros engenheiros de frontend, que trabalham a anos nessa área, jamais e nem em um milhão de anos saberão como construir um website "do caralho" como este cara aqui ensina para você. :smile: 

http://www.websitedocaralho.com.br

(sério moderador nao apaga)

**Comment by felipefialho** [comment]
hahaha :laughing:

**Comment by icaromh** [comment]
Tem uma outra versão também mas o cara não ajustou as referências :-1: 

http://tatudobem.blog.br/websitemaisdocaralhoainda/

**Comment by bernardodiasc** [comment]
haha esse segundo é do caralho tambem. content is the fucking king.

**Comment by notnotgabriel** [comment]
Isso é genial :laughing:

**Comment by bernardodiasc** [comment]
http://jgthms.com/web-design-in-4-minutes/ um pouco similar, muito interessante.
